data/json/monsters/zed_misc.json

```diff
@@ -795,6 +805,8 @@
     "armor_elec": 1,
     "bleed_rate": 50,
     "vision_night": 5,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
```

Comment:
Probably should have a more advanced weakpoint set if it barely looks human.

Reply:
What do you think about this one?